server/datastore/mysql/scripts.go

```diff
@@ -128,7 +128,7 @@ func (ds *Datastore) SetHostScriptExecutionResult(ctx context.Context, result *f
 		res, err := tx.ExecContext(ctx, updStmt,
 			output,
 			result.Runtime,
-			result.ExitCode,
+			int32(result.ExitCode),
```

Comment:
Nit but I don't think this is necessary.

Reply:
Oh I see that it might be just for the test case (otherwise the cast to signed happens in fleetd, not server-side). That's fine to leave it there but if you have a sec to add a comment as for the why of this, as it's the kind of thing someone may see and have a reflex to remove as unnecessary.

Reply:
I'd be very happy to put a comment 🙂

Reply:
I added it to the server as well because I wanted to make sure the bug gets fixed for clients where orbit can't be updated for some reason

Reply:
Would you prefer I remove the server-side overflow check and just rely on the client?

Reply:
No, your rationale makes sense to me!